app/helpers/form_helper.rb

```diff
@@ -94,4 +94,8 @@ def international_phone_codes_data(code_data)
       country_name: code_data['name'],
     }
   end
+
+  def idp_simple_form_for(record, options = {}, &block)
+    simple_form_for(record, options, &block) + javascript_pack_tag('simple-form-validation')
```

Comment:
I guess this could be simpler?

```suggestion
  def idp_simple_form_for(*args, &block)
    simple_form_for(*args, &block) + javascript_pack_tag('simple-form-validation')
```

Not sure which is clearer in terms of:

- Passing through arguments as given
- Passing through arguments explicitly

Reply:
I like the suggestion, since we're not touching the args at all in this method

Reply:
> I like the suggestion, since we're not touching the args at all in this method

Updated in 839100a.